dev/dfi/packet.c

```diff
@@ -161,6 +412,229 @@ void dfi_rx_packet_buffer_data_compare(dfi_rx_packet_buffer_t *buffer,
     *is_same = same;
 }
 
+static void packet_set_wrdata_all_phases(packet_item_t *packet, chipselect_t cs, uint8_t vref_setting)
```

Comment:
let's add cbt to the function name so it's clear that it's only to be used with CBT.

Reply:
This function was removed, now uses the fill_wrdata_packet function api.